Lib/test/test_sqlite3/test_dbapi.py

```diff
@@ -650,6 +665,14 @@ def __getitem__(slf, x):
         with self.assertRaises(ZeroDivisionError):
             self.cu.execute("select name from test where name=?", L())
 
+    def test_execute_too_many_params(self):
+        category = sqlite.SQLITE_LIMIT_VARIABLE_NUMBER
+        msg = "too many SQL variables"
+        with cx_limit(self.cx, category=category, limit=1):
+            self.cu.execute("select abs(?)", (-1,))
+            with self.assertRaisesRegex(sqlite.OperationalError, msg):
+                self.cu.execute("select max(?, ?)", (1, 2))
```

Comment:
It will also raise OperationalError if the SQLITE_LIMIT_FUNCTION_ARG limits is set to 1. I think the former example was simpler.